README.md

```diff
@@ -344,20 +346,21 @@ we publish an Apache Airflow release. Those images contain:
 
 The version of the base OS image is the stable version of Debian. Airflow supports using all currently active
 stable versions - as soon as all Airflow dependencies support building, and we set up the CI pipeline for
-building and testing the OS version. Approximately 6 months before the end-of-life of a previous stable
-version of the OS, Airflow switches the images released to use the latest supported version of the OS.
+building and testing the OS version. Approximately 6 months before the end-of-regular support of a
+previous stable version of the OS, Airflow switches the images released to use the latest supported
+version of the OS.
+
 For example since ``Debian Buster`` end-of-life was August 2022, Airflow switched the images in `main` branch
 to use ``Debian Bullseye`` in February/March 2022. The version was used in the next MINOR release after
 the switch happened. In case of the Bullseye switch - 2.3.0 version used ``Debian Bullseye``.
 The images released  in the previous MINOR version continue to use the version that all other releases
-for the MINOR version used.
-
-Support for ``Debian Buster`` image was dropped in August 2022 completely and everyone is expected to
-stop building their images using ``Debian Buster``.
+for the MINOR version used. Similar switch from ``Debian Bullseye`` to ``Debian Bookworm`` has been implemented
+before 2.8.0 release in October 2023 and ``Debian Bookworm`` will be the only option supported as of
```

Comment:
I am slightly confused on when do we plan to make the switch. Could it be before 2.8.0 or it will done with 2.8.0? 
also can we please confirm the "October 2023" here is what we intend to mention?